app/Http/Controllers/ResultsController.php

```diff
@@ -66,7 +66,8 @@ public function index(MismatchGetRequest $request, WikibaseAPIClient $wikidata):
                 'user' => $user,
                 'item_ids' => $requestedItemIds,
                 // Use wikidata to fetch labels for found entity ids
-                'labels' => $wikidata->getLabels($entityIds, $lang),
+                // array_filter removes empty values from an array if no callback argument is passed
```

Comment:
This comment should probably not be here, it adds no additional information and is prone to get outdated quickly.

(I'd probably also argue against the comment line directly above, but that was not added by this PR, so that's out of scope here.)

Reply:
`array_filter` is well documented at the PHP manual and can easily be looked up there: https://www.php.net/manual/en/function.array-filter.php

The trouble with such comments is that they are duplicating what is already written in the code itself and tend to get overlooked and out of date when someone changes the code.

Here is a bit more about how to fix such an issue: https://refactoring.guru/smells/comments

Though in this case, I would still argue that the comment can just be dropped, because it merely explains the default behavior of a core PHP function.

Reply:
I thought it might help, but if it doesn't then yes. :) Removed.